docs/configuration_options.md

```diff
@@ -1,2737 +1,3525 @@
 # scylla-cluster-tests configuration options
 
+#### Appending with environment variables or with config files
+* **strings:** can be appended with adding `++` at the beginning of the string:
+`export SCT_APPEND_SCYLLA_ARGS="++ --overprovisioned 1"`
+* **list:** can be appended by adding `++` as the first item of the list
+`export SCT_SCYLLA_D_OVERRIDES_FILES='["++", "extra_file/scylla.d/io.conf"]'`
+
 ## **config_files** / SCT_CONFIG_FILES
 
 a list of config files that would be used
 
 **default:** N/A
 
+**type:** str_or_list_or_eval
+
 
 ## **cluster_backend** / SCT_CLUSTER_BACKEND
 
 backend that will be used, aws/gce/docker
 
 **default:** N/A
 
+**type:** str
+
 
 ## **test_method** / SCT_TEST_METHOD
 
 class.method used to run the test. Filled automatically with run-test sct command.
 
 **default:** N/A
 
+**type:** str
+
 
 ## **test_duration** / SCT_TEST_DURATION
 
 Test duration (min). Parameter used to keep instances produced by tests<br>and for jenkins pipeline timeout and TimoutThread.
 
 **default:** 60
 
+**type:** int
+
 
 ## **prepare_stress_duration** / SCT_PREPARE_STRESS_DURATION
 
 Time in minutes, which is required to run prepare stress commands<br>defined in prepare_*_cmd for dataset generation, and is used in<br>test duration calculation
 
 **default:** 300
 
+**type:** int
+
 
 ## **stress_duration** / SCT_STRESS_DURATION
 
 Time in minutes, Time of execution for stress commands from stress_cmd parameters<br>and is used in test duration calculation
 
 **default:** N/A
 
+**type:** int
+
 
 ## **n_db_nodes** / SCT_N_DB_NODES
 
 Number list of database data nodes in multiple data centers. To use with<br>multi data centers and zero nodes, dc with zero-nodes only should be set as 0,<br>ex. "3 3 0".
 
 **default:** N/A
 
+**type:** int_or_list (appendable)
```

Comment:
In the code it is called `int_or_list`, yes.
But practically, we can set `int` or string like `3 3 3`.

FOr now I find it to be confusing, because if I wouldn't know, I would try to use list object for multi-dc cases.

Reply:
you have a point, according to this docs

we should be able to do:
```
export SCT_N_DB_NODES='[3, 2, 3]'
```

but the code doesn't really expect a list from those, code can only handle `int` or `str`

this function should be renamed to `int_or_space_seperated_ints` :), and remove the ast.eval from it.

but that is confusing regardless of the PR that the situation.

we probably should have those values appendable, cause it's not completely defined (i.e. how do you append to int ?), those should be overridable only

Reply:
I introduce that `ast.eval`, back in https://github.com/scylladb/scylla-cluster-tests/pull/1099

which is one of the first multi-dc we stabilized after the move from avocado configuration  to sct_config
I don't see any real reason for it, but I'm guessing I wanted to back out from the space separated values.

backing from it now, is a bit more tricky and would need to change lots of places that those it.

Reply:
anyhow handling this in https://github.com/scylladb/scylla-cluster-tests/pull/9602, it's not for this PR